securesystemslib/keys.py

```diff
@@ -1637,6 +1726,46 @@ def import_ed25519key_from_private_json(json_str, password=None):
   return key_object
 
 
+def import_spxkey_from_private_json(json_str, password=None):
+  if password is not None:
+    # This check will not fail, because a mal-formatted passed password fails
+    # above and an entered password will always be a string (see get_password)
+    # However, we include it in case PASSWORD_SCHEMA or get_password changes.
+    securesystemslib.formats.PASSWORD_SCHEMA.check_match(password)
+
+    # Decrypt the loaded key file, calling the 'cryptography' library to
+    # generate the derived encryption key from 'password'.  Raise
+    # 'securesystemslib.exceptions.CryptoError' if the decryption fails.
+    key_object = securesystemslib.keys.\
+                 decrypt_key(json_str.decode('utf-8'), password)
+
+  else:
+    logger.debug('No password was given. Attempting to import an'
+        ' unencrypted file.')
+    try:
+      key_object = \
+               securesystemslib.util.load_json_string(json_str.decode('utf-8'))
+    # If the JSON could not be decoded, it is very likely, but not necessarily,
+    # due to a non-empty password.
+    except securesystemslib.exceptions.Error:
+      raise securesystemslib.exceptions\
```

Comment:
How come you broke the line before the "." instead of after like in previous statements?